### ISSUE TITLE
Missing Spec Challenge

### ISSUE THREAD

**Issue by imsrybr0** [Certora/tutorials-code#7]
- PR : https://github.com/Certora/tutorials-code/pull/6
- `BordaMissingRule.spec` on the original `Borda.sol` : https://prover.certora.com/output/33434/d7f15af5748b43a38eebb1e07f1ba79e/?anonymousKey=8799eb540328477c469b5c8584b792b10ef9af32
- `Borda.spec` on `BordaNewBug.sol` : https://prover.certora.com/output/33434/ad728d166f6e4ae6a9d144407c7eae65/?anonymousKey=6f6735dba598be65de68b3c84d935a4706205331
- `BordaMissingRule.spec` on `BordaNewBug.sol` : https://prover.certora.com/output/33434/bee54e17460f471fba4613b08294725d/?anonymousKey=3dbed3eabc5884bea4c52da4b59eda0730d79f44

**Comment by Czar102** [comment]
That's a cool idea, but the spec [doesn't pass for me](https://prover.certora.com/output/2949/5c5dfc029bb647aea8c2dd3e0cd998b0?anonymousKey=20fe82670df2192137fd15678aebd8e6e81a2648) for the original `Borda` contract (in case above a sanity check failed, those happen weirdly too often with advanced sanity checks…)

**Comment by imsrybr0** [comment]
You're right, my bad, I ended pushing a version of the spec where I was testing a bidirectional assertion. Can you try it again with one direction ?

**Comment by Czar102** [comment]
Because this didn't work for me, I made #12 instead. Would you agree that it's more general?

**Comment by imsrybr0** [comment]
Both of the rules above work (you can check the runs for more details). Can you share links to runs where it didn't work for you?

As for your rule set, I tried it and it seems to catch the bug introduced by my mutation as well :+1:

**Comment by nd-certora** [comment]
Nice work, this is missing spec #2.
